### ISSUE TITLE
Where are the Haskell communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/haskell#684]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Haskell supported by one or more large organizations?
* Does Haskell have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Haskell? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Haskell? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Haskell? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Haskell?
* Are there popular newsletters for Haskell?
* Is Haskell taught at programming bootcamps? (If so, what are some examples?)
* Is Haskell taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Haskell?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by rpearce** [comment]
> Do you know of specific communities (online or offline) that are enthusiastic about Haskell? (Chat communities, forums, meetups, student clubs, etc)
* https://functionalprogramming.slack.com (numerous FP channels on there and a number of great Haskell ones)

> Are there popular conferences for Haskell? (If so, what are some examples?)
* http://www.lambdadays.org
* http://lambdaconf.us

> Are there popular newsletters for Haskell?
* https://haskellweekly.news

**Comment by Average-user** [comment]
The #Haskell IRC channel on Freenode is pretty active too

**Comment by ajkavanagh** [comment]
Reddit: https://www.reddit.com/r/haskell/

University courses (freely available):

* NICTA: https://github.com/~-Haskell-course                                                                           
* Data62 Functional Programming Course (Formerly NICTA): https://github.com/~/fp-course                                
* CIS 194: Introduction to Haskell (2016): https://www.seas.upenn.edu/~/fall16/

IRC: freenode: #159 
All things haskell: https://wiki.haskell.org/Haskell

* Microsoft employs Simon Peyton-Jones as a researcher; he's one of the founders of the language.
* Facebook employs Simon Marlow; key developer of the GHC (compiler for Haskell)

* Haskell community information: https://haskell-lang.org/community

**Comment by rpearce** [comment]
@ajkavanagh None of your `University courses` links are working. Can you double-check them and update accordingly? Many thanks <3

**Comment by ajkavanagh** [comment]
@rpearce sorry about that (copy/paste error from vimwiki!)  -- sorted out now.

**Comment by kytrinyx** [comment]
This is great information, thank you!
